projects/storefrontlib/cms-components/content/tab/panel/tab-panel.component.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Input } from '@angular/core';
+import { Tab, TAB_MODE } from '../tab.model';
+
+@Component({
+  selector: 'cx-tab-panel',
+  templateUrl: './tab-panel.component.html',
+})
+export class TabPanelComponent {
+  TAB_MODE = TAB_MODE;
+
+  /**
+   * Tab object to display content and set attributes to.
+   */
+  @Input() tab: Tab;
+
+  @Input() mode: TAB_MODE;
```

Comment:
Providing a default value to avoid undefined? such as @Input() mode: TAB_MODE = TAB_MODE.TAB